py/desiutil/test/test_install.py

```diff
@@ -35,7 +35,8 @@ class TestInstall(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        pass
+        cls.py = f"python{sys.version_info.major:d}.{sys.version_info.minor:d}"
```

Comment:
Minor: since we also have a directory called "py", I suggest that this variable name be called "pyver" or "pyversion" or similar to disambiguate the meaning when it is used later.

Reply:
Done